client/src/components/CreateInvoiceForm/index.js

```diff
@@ -274,9 +300,9 @@ export default class index extends Component {
           </section>
           <section className="mid-section">
             <div className="mid-section-left">
-              <div>
+              <div className="address-from">
                 <form onSubmit={this.handleFormSubmit}>
-                  <div>From</div>
+                  <div>FROM</div>
                   <TextArea
                     inputType={"text"}
                     //title={"Invoice From"}
```

Comment:
Code looks good! Is this something that going to be not used or just commented out until the Invoice Form is ready ?